src/test/java/bio/terra/app/usermetrics/UserMetricsInterceptorTest.java

```diff
@@ -109,7 +109,7 @@ public void testSendEventNotFiredWithNoToken() throws Exception {
         .when(authenticatedUserRequestFactory)
         .from(any());
 
-    runAnWait(request, response);
+    runAndWait(request, response);
 
     verify(bardClient, times(0)).logEvent(any(), any());
```

Comment:
If you use strict stubbing, you shouldn't need to `verify(... times(0))` since the strict stubs check does that for you.

Another possibility is to use `verifyNoMoreInteractions()`, which I think has the same effect.